torch_xla/csrc/tensor.h

```diff
@@ -467,6 +467,9 @@ class XLATensor {
 
   static XLATensor ceil(const XLATensor& input);
 
+  static XLATensor celu(const XLATensor& input, const at::Scalar& alpha);
```

Comment:
I think the goal of this pr is to get rid of `elu`, do you plan to remove that in a separate pr?

Reply:
Yep, once this merges I'll rebase https://github.com/pytorch/xla/pull/3539 which is currently failing due to `celu` and `selu` tests. Then we should be able to get rid of the scope pusher and IR level computation that is being done with `elu`.